locale/00_UpdateLocalization.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../RubySetupSystem/RubyCommon'
```

Comment:
The script should start with
```
#!/usr/bin/env ruby
# frozen_string_literal: true
```
and the file should have the executable bit set (might be hard to do on Windows)